src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -4569,6 +4569,10 @@ Copyright (C) Microsoft Corporation. All rights reserved.
       <Output TaskParameter="TargetOutputs" ItemName="_ThisProjectItemsToCopyToOutputDirectory" />
     </CallTarget>
 
+    <ItemGroup Condition="'$(IgnoreConflictingTransitiveContent)' == 'true'">
```

Comment:
Wait, why true? Shouldn't it be false? If it is true that we are ignoring conflicts, we should not be doing anything, i.e., we should not remove anything from _TransitiveItemsToCopyToOutputDirectory.

Reply:
Note that whether it's true or false doesn't affect whether it's opt-in or opt-out, since the default is nothing.

Reply:
My bad, I misread the name of the property. I've changed it so the condition is `IgnoreConflictingTransitiveContent == false`

Reply:
Also, have you prepped a change in the SDK to set IgnoreConflictingTransitiveContent?

Reply:
I haven't prepped the change, feel free to make it if you want. I don't think you would want to put the property in Directory.Build.props, though, since I believe that's only used for building the SDK itself. Instead, we want it in one of [the .props files that ships with the SDK](https://github.com/dotnet/sdk/tree/master/src/Tasks/Microsoft.NET.Build.Tasks/targets). I would put it in [Microsoft.NET.Sdk.props](https://github.com/dotnet/sdk/blob/master/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.props#L105)

Reply:
I put it where you suggested, but I don't fully understand the implications. I thought both would be included in any SDK-style project by default, so putting it in either places would have the same effect?

Reply:
Putting it in any of [these files](https://github.com/dotnet/sdk/tree/master/src/Tasks/Microsoft.NET.Build.Tasks/targets) would have the same effect, since all of them are included in the SDK by default, so among those it's just convention. However, the original link pointed to the Directory.Build.props file at the root of the repo, which isn't included in the SDK as it's used to customize our build (note that we do some things that are specific to building the SDK in that file like import arcade and set warnings as errors).